bundles/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudClient.java

```diff
@@ -684,4 +694,11 @@ private void sleepSocketIO(long delay) {
             }
         });
     }
+
+    private static String sensored(String secret) {
```

Comment:
I think you mean censored?

Reply:
https://github.com/openhab/openhab-addons/pull/12430/commits/fc8043adb2d62c13664503d49ac7e5c0910c6aac